### ISSUE TITLE
[iOS][Accessibility] [View is getting scrolled on submit validation for input.text and input.number when voice over is on]

### ISSUE THREAD

**Issue by jyoti-ms** [microsoft/AdaptiveCards#5024]
# Platform

- [ ]  iOS

# Author or host
Teams

# Version of SDK
2.3.0

# Details
View is getting scrolled and hence, focus is not going to the accessibility element when we have input.text / input.number as the first failed input.
